seminars/practice_6_classes.py

```diff
@@ -1,269 +0,0 @@
-"""
-Programming 2024
-Seminar 6
-
```

Comment:
please revert this change

Reply:
!!!

Reply:
![image](https://github.com/user-attachments/assets/c88b2d4a-dca1-4450-9abe-86abe377f70d)

Reply:
![image](https://github.com/user-attachments/assets/43ddfb47-1c2b-4a27-aa5b-ee7f3f326210)

Fixed